full-stack-tests/rpc-interface/src/frontend/setup/TestContext.ts

```diff
@@ -61,15 +60,12 @@ export class TestContext {
     Logger.setLevelDefault(this.settings.logLevel === undefined ? LogLevel.Warning : this.settings.logLevel);
 
     if (undefined !== this.settings.oidcClientId) {
-      this.adminUserAccessToken = await getAccessTokenFromBackend({
-        email: this.settings.users[0].email,
-        password: this.settings.users[0].password,
-      } as TestUserCredentials, {
+      this.serviceAuthToken = await getServiceAuthTokenFromBackend({
         clientId: this.settings.oidcClientId,
```

Comment:
i'm pretty sure the exsiting oidc clientId will be from a non service client no? So it shouldn't have a client secret, and the tests would fail?

Reply:
yes you are right, just reassigned values to the variables to make it a frontend client